lib/stacker/stack/capabilities.rb

```diff
@@ -9,7 +9,10 @@ def local
       end
 
       def remote
-        @remote ||= client.capabilities
+        # `capabilities` actually returns a
+        # !ruby/array:Aws::Xml::DefaultList, so we convert to a Ruby
+        # Array with `map`
+        @remote ||= client.capabilities.map(&:self)
```

Comment:
I believe `to_a` is the idiomatic way to accomplish this. It returns `Array` if subclassed.